Procfile

```diff
@@ -0,0 +1,2 @@
+web: ./docker/run-prod.sh
```

Comment:
This will work on its own for a new app, but we've seen issues switching from "cmd" process type to the "web" proc type when deploying running apps. I wonder if it would be safer to `s/web/cmd/`?